index.html

```diff
@@ -3,449 +3,298 @@
+        out from under them in incompatible ways.
+      </p>
     </section>
     <section>
       <h2>
         Introduction
       </h2>
       <p>
         Web applications often accept developer-provided image resources to be
-        displayed. This specification defines a generalized structure where
-        developers can provide additional metadata about an image. The user
-        agent can then use this extra information to choose an an image that is
-        best suited to display on the end-user's device or most closely matches
-        the end-user's preferences.
+        displayed outside of a HTML document (e.g., in the OS, in the browser
+        UI, etc.). This specification defines a WebIDL dictionary that
+        describes an image, and from where that image resource can be fetched
+        from. The user agent can then use this extra information to choose an
+        an image that is best suited to display on the end-user's device or
+        most closely matches the end-user's preferences or environment.
       </p>
     </section>
     <section>
       <h2>
-        Infrastructure
+        Image resource
       </h2>
-      <section>
-        <h3>
-          Processed image resource
-        </h3>A <dfn>processed image resource</dfn> consists of:
-        <dl>
-          <dt>
-            <dfn data-dfn-for="processed image resource">src</dfn>
-          </dt>
-          <dd>
-            A <a>URL</a>.
-          </dd>
-          <dt>
-            <dfn data-dfn-for="processed image resource">sizes</dfn>
-          </dt>
-          <dd>
-            A <a>set</a> of <a>strings</a>.
-          </dd>
-          <dt>
-            <dfn data-dfn-for="processed image resource">type</dfn>
-          </dt>
-          <dd>
-            A <a>string</a>.
-          </dd>
-        </dl>
-      </section>
+      <p>
+        An <dfn>image resource</dfn> consists of:
+      </p>
+      <dl>
+        <dt>
+          <dfn data-dfn-for="image resource">src</dfn>
+        </dt>
+        <dd>
+          A [=URL=] from where to obtain the image data.
+        </dd>
+        <dt>
+          <dfn data-dfn-for="image resource">sizes</dfn>
+        </dt>
+        <dd>
+          A [=string=] representing the sizes of the image, expressed using the
+          same syntax as [^link^]'s [^link/sizes^] attribute.
+        </dd>
+        <dt>
+          <dfn data-dfn-for="image resource">type</dfn>
+        </dt>
+        <dd>
+          An <a>image MIME type</a>.
+        </dd>
+      </dl>
+      <p>
+        In an API, this is represented as a {{ImageResource}} dictionary.
+      </p>
     </section>
-    <section>
+    <section data-dfn-for="ImageResource">
       <h2>
-        <dfn>ImageResource</dfn> and its members
+        <dfn>ImageResource</dfn> dictionary
       </h2>
       <pre class="idl">
         dictionary ImageResource {
           required USVString src;
           DOMString sizes;
-          USVString type;
+          DOMString type;
```

Comment:
The manifest spec currently defines this as a USVString. Is this change intentional?

Reply:
Yes. I can't think of a reason  to make this would be an USVString?